src/psyclone/nemo.py

```diff
@@ -198,6 +214,12 @@ def __init__(self, ast):
         # Keep a pointer to the whole fparser2 AST
         self._ast = ast
 
+        # Use the fparser2 frontend to construct the PSyIR from the parse tree
+        processor = NemoFparser2Reader()
+        # First attempt to create a Container representing any Fortran module
```

Comment:
Remove `attempt to` as there is no exception logic here

Reply:
Oh yes, done.